ch_pipeline/hfb/analysis.py

```diff
@@ -83,11 +83,11 @@ def process(self, stream):
             # Number of samples with non-zero weight in each time bin
             nsamples = np.sum(binary_weight, axis=axis_index)
 
-            # For uniform weighting, the average of the variances is
-            # sum( 1 / weight ) / nsamples,
-            # and the averaged weight is the inverse of that
+            # For uniform weighting, the variance of averaged data is
+            # sum( variance ) / (nsamples**2),
+            # and the weight is the inverse of that
             variance = tools.invert_no_zero(weight)
-            avg_weight = nsamples * tools.invert_no_zero(
+            avg_weight = (nsamples**2) * tools.invert_no_zero(
```

Comment:
```suggestion
            avg_weight = nsamples**2 * tools.invert_no_zero(
```
I think the parentheses are not needed